marklogic-client-api/src/main/java/com/marklogic/client/DatabaseClientFactory.java

```diff
@@ -325,6 +325,13 @@ public interface SecurityContext {
      */
     SSLContext getSSLContext();
 
+	  /**
+	   * @return the trust manager for this connection
+	   * @since 6.1.0 ; was already implemented by both implementations of this interface, and exposing it here
```

Comment:
For each new public addition going forward, we're going to use `@since` to make life easier for all of us.